ops/prod/persistent/okta.tf

```diff
@@ -3,6 +3,8 @@ module "okta" {
   env                  = local.env
   app_url              = "https://www.simplereport.gov/app"
   logout_redirect_uris = ["https://simplereport.cdc.gov", "https://www.simplereport.gov"]
+  trusted_origin_urls = [["SimpleReport Prod Env", "https://simplereport.gov", ["CORS, REDIRECT"]],
```

Comment:
I'm only seeing URLs for `stg` and `prod`, but we'll need one for every environment I think?

Reply:
I pulled what already existed in the list. We can certainly add more, but note that they are not currently accounted for. (Plus, Okta isn't leveraged in `demo`, so that's one to leave out.)

Reply:
My bad, I just checked the list and saw what you see. This looks good then.